jdaviz/configs/default/plugins/gaussian_smooth/tests/test_gaussian_smooth.py

```diff
@@ -120,7 +120,7 @@ def test_spatial_convolution(cubeviz_helper, spectrum1d_cube):
             == (2, 4, 2))
 
 
-def test_spectrum1d_smooth(specviz_helper, spectrum1d):
+def test_specviz1d_smooth(specviz_helper, spectrum1d):
```

Comment:
If you want to rename to match the viz tool, dropping "1d" would be less confusing.
```suggestion
def test_specviz_smooth(specviz_helper, spectrum1d):
```